peers/validators/canonical_validator_client.go

```diff
@@ -89,86 +81,21 @@ func (v *CanonicalValidatorClient) GetCurrentValidatorSet(
 }
 
 // Gets the validator set of the given subnet at the given P-chain block height.
-// Attempts to use the "getValidatorsAt" API first. If not available, falls back
-// to use "getCurrentValidators", ignoring the specified P-chain block height.
+// Uses [platform.getValidatorsAt] with "proposed" height to get the validator set that
+// will likely be used for the next block.
```

Comment:
This comment doesn't match the implementation, which uses the passed in height

Reply:
Good catch! Will update